Gemfile

```diff
@@ -80,6 +80,7 @@ gem 'webauthn', '~> 2.5.2'
 gem 'xmldsig', '~> 0.6'
 gem 'xmlenc', '~> 0.7', '>= 0.7.1'
 gem 'yard', require: false
+gem 'zlib', require: false
```

Comment:
Did we really not have this already? Or had we pulled it in for Attempts API stuff and then removed it?

No objection to including it; I just would have sworn it was already here.

Reply:
Somehow not!
```bash
> git log origin/main -Szlib -- Gemfile.lock
# empty results
```